change/@fluentui-react-examples-3f0c2d26-ca9c-4677-8047-c4b0293b1fff.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Renaming react-cards package to react-card (singular)",
+  "packageName": "@fluentui/react-examples",
+  "email": "czearing@outlook.com",
+  "dependentChangeType": "none"
```

This should be a patch change.
```suggestion
  "type": "patch",
  "comment": "Renaming react-cards package to react-card (singular)",
  "packageName": "@fluentui/react-examples",
  "email": "czearing@outlook.com",
  "dependentChangeType": "patch"
```